docs/schema.md

```diff
@@ -223,6 +223,23 @@ Explanation of what the property is or what is used for. Be mindful of good expl
 
 If user does not provide a value, `default` will be used.
 
+### enum
+
+Array property, that will create combo box with a list of provided values, instead of free form input field.
+
+```yaml
+  some.property:
+    type: string
```

Comment:
Maybe it's worth to add that other types are also supported.

Reply:
I'm not sure - this is an example how to use enum, other parts of documentation are not type specific (eg. `minimum` and `maximum`). To keep the same style in `enum`, I prefer not to add type explicitly for only this property. I think it would look strange.